VSTS/Tasks/SonarQubeScannerCli/SonarQubeScanner.ps1

```diff
@@ -0,0 +1,98 @@
+[CmdletBinding(DefaultParameterSetName = 'None')]
```

Comment:
@bgavrilMS We will rewrite the tasks in typescript in a future sprint. For now (v1) we just go with powershell but thx for the reminder :)